api/GL/glcorearb.h

```diff
@@ -3838,6 +3838,12 @@ GLAPI void APIENTRY glTexBufferARB (GLenum target, GLenum internalformat, GLuint
 
 #ifndef GL_ARB_viewport_array
 #define GL_ARB_viewport_array 1
+typedef void (APIENTRYP PFNGLDEPTHRANGEARRAYDVNVPROC) (GLuint first, GLsizei count, const GLdouble *v);
```

Comment:
These are unrelated to my changes. It looks like there's a recent commit that didn't regenerate the headers.